site/static/diagrams/gameserver-states.dot

```diff
@@ -23,9 +24,14 @@ digraph {
     PortAllocation -> Creating [ label="allocated unused port", color="blue" ]
     Creating -> Starting [ label="created pod", color="blue" ]
     Starting -> Scheduled [ label="we have a pod, fetch its address", color="blue" ]
+    Scheduled -> Reserved [ label="SDK.reserved(seconds)", color="purple" ]
     Scheduled -> RequestReady [ label="SDK.ready()", color="purple" ]
     RequestReady -> Ready [ label="ready to be allocated", color="blue" ]
+    Reserved -> Ready [label="if seconds > 0  \nand failed to call  \nSDK.allocate()  " color="purple"]
+
     Ready -> Allocated [ label="allocated for use", color="orange" ]
+    Ready -> Allocated [ label="SDK.allocate()   ", color="purple" ]
+    Reserved -> Allocated [color="purple"]
```

Comment:
There isn't a line from Reserved -> Shutdown. It seems like there is a state transition from every state to both Unhealthy and to Shutdown, but this one is missing (shutdown only has 4 incoming arrows and unhealthy has 5).